src/components/Footer.jsx

```diff
@@ -1,20 +1,175 @@
 import React from 'react'
+import { Link } from 'gatsby'
+import slackLogo from '../images/slack.svg'
+import githubLogo from '../images/github.svg'
+import meetup from '../images/meetup.svg'
 import styled from 'styled-components'
-import logo from '../images/cfc_logo_short.svg'
+import longArrowAltRight from '../images/join-us/long-arrow-alt-right.svg'
 
 const StyledFooter = styled.footer`
-  width: 14rem;
+  width: 100%;
+  padding: 1rem 0;
+  display: block;
+  margin: 0 auto;
+
+  @media (min-width: 1200px) {
+    padding: 0rem 15.25rem;
+  }
+
+  .footer-content {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+
+    @media (min-width: 1200px) {
+      width: 653.16px;
+    }
+  }
+
+  .footer-links {
```

Comment:
Might have a better time adding `gap` 1rem to this class and clearing the margins on the individual other children. Looks like it's not consistent with the Figma at the mid breakpoint. 
<img width="581" alt="Screen Shot 2023-01-15 at 7 24 23 PM" src="https://user-images.githubusercontent.com/6326891/212579794-f5b8fe33-c45b-4f7f-aad1-328569b8a2c0.png">